src/Endpoints/Reports/AccountListingWithListingPrice.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Hitmeister\Component\Api\Endpoints\Reports;
+
+class AccountListingWithListingPrice extends Post
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getURI()
+	{
+		return 'reports/account-listing-with-listing-price/';
+	}
+}
```

Comment:
Please add a new line at the end of this file

Reply:
added new line at the end